src/main/java/com/ksm/wstbackoffice/service/CountryService.java

```diff
@@ -21,6 +22,7 @@ public CountryService(CountryRepository countryRepository, CountryMapper country
     }
 
     public CountryDto findById(String id) {
-        return countryMapper.toDto(countryRepository.findById(id).orElse(null));
+        return countryMapper.toDto(countryRepository.findById(id)
+                .orElseThrow(() -> new ResourceNotFoundException("Not found country with id = " + id)));
```

Comment:
It is better in this case instead of 
`
"Not found country with id = " + id
`
to use
String.format("Not found country with id %s ", id)

Reply:
done

Reply:
Probably you forgot to push your changes. Because it is not fixed regardless you wrote "done"